### PR TITLE
3.26 support

### PR DESCRIPTION
**Pull request by tanty**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -64,7 +66,6 @@ develop:
 dist: clean-build collect compile
 # We need to do this like this as 'zip' always uses the cwd as archive root.
 # And for the extension to work extension.js etc. need to be at the root.
-	mkdir -p $(BUILDDIR);
```
Comment (elbenfreund) [inline Makefile]: Do you remove this line because ``collect`` is already taking care of creating this dir?

**Comment by elbenfreund** [comment]
@tanty ping :)

File: Makefile (hunk 1 above)
Comment (tanty) [reply Makefile]: Yes. The target depends on `collect` and `collect` is already doing this. Therefore, it is redundant.

**Comment by tanty** [comment]
@elbenfreund answered. Sorry for the delay and thanks for this great extension! :smile:

**Comment by elbenfreund** [comment]
#221 and #223 have been merged. Thanks again for your help!

**Pull request closed without merging**